src/meson.build

```
@@ -5,7 +5,9 @@ blueprint_compiler = find_program(
   # Flatpak
   '/app/bin/blueprint-compiler',
   # host
-  '../blueprint-compiler/blueprint-compiler.py'
+  '../blueprint-compiler/blueprint-compiler.py',
+  # host (installed on system)
+  '/usr/bin/blueprint-compiler'
```

```suggestion
  # host
  'blueprint-compiler'
```

but I'm not convinced it's a good idea – blueprint is still a moving target and that's why it's included in Tangram (as a git submodule)

what is your use case?

---

I made a package for Tangram in Alpine Linux, and since we already package blueprint-compiler I figured I'd make it use the system compiler instead of the submodule. Just submitted this because I figured it could be useful for any other users of distros with packaged blueprint-compiler; given that not all distros might update it as regularily as needed, and it could cause weird compilation errors, I'm completely fine with leaving this as a downstream patch only.

---

:+1:  Appreciate that Alpine always sends patch upstream.